miasm2/core/asmblock.py

```diff
@@ -91,7 +53,7 @@ class AsmConstraint(object):
 
     def __init__(self, label, c_t=c_to):
         # Sanity check
-        assert isinstance(label, AsmLabel)
+        assert isinstance(label, LocKey)
 
         self.label = label
```

Comment:
`.label` is actually a `loc_key` now. Maybe a property with a deprecation warning?

Reply:
done